### ISSUE TITLE
Setup outputs as str

### ISSUE THREAD

**Issue by vfdev-5** [stared/livelossplot#113]
## 🚀 Feature

Just to improve user's experience with this part of code:

- Currently
```python
from livelossplot import PlotLosses
from livelossplot.outputs import MatplotlibPlot, TensorboardLogger

outputs=[MatplotlibPlot(), TensorboardLogger()]
plotlosses = PlotLosses(outputs=outputs)
```

- FR
```python
from livelossplot import PlotLosses

outputs=["matplotlib", "tensorboard", ...]
plotlosses = PlotLosses(outputs=outputs)
```

@stared what do you think ?

**Comment by stared** [comment]
I considered that, but prefer to avoid extraneous syntax. Especially in a situation when if we want custom outputs, usually we want custom outputs (e.g. special path to `TensorboardLogger` ).

If the problem is with extra import line, would the following (not yet implemented) work for you:

```{python}
from livelossplot import PlotLosses

plotlosses = PlotLosses(outputs=[])
plotlosses.add_matplotlib()
plotlosses.add_tensorboard()
```

**Comment by vfdev-5** [comment]
OK, I see. Maybe, either like this:
```python
from livelossplot import PlotLosses

plotlosses = PlotLosses().add_matplotlib().add_tensorboard()
```

**Comment by stared** [comment]
I am a big fan of chaining, so I like it.

Now I see that `.to_` may be more descriptive than `.add_`.

**Comment by stared** [comment]
@vfdev-5 Merged (will be released with `livelossplot >0.5.2`):

* outputs as strings, e.g. `PlotLosses(outputs=['MatplotlibPlot', 'TensorboardLogger'])` in case we use default parameters
* `PlotLosses(outputs=[]).to_matplotlib(...).to_tensorboard(...)` if we want to use custom parameters but don't want an extra import line

**Comment by vfdev-5** [comment]
Looks nice, thanks !

**Comment by stared** [comment]
OK, we released `0.5.3`.
